.golangci.yml

```diff
@@ -24,7 +24,6 @@ linters:
     - gocritic
     - gofmt
     - goimports
-    - gosec
```

Comment:
Why were these bits removed?

Reply:
It complains about any URLs in the provider files that don't start with https://. I was going to add another exclusion but didn't think it was helpful.

Reply:
I added it back in and made another exclusion rule.